submissions/LuckyDnepr/task-js-oop/app.js

```diff
@@ -0,0 +1,173 @@
+"use strict";
+
+const blockWidth = 101, //gamefield sizes
+    blockHeight = 83,
+    nBlocksWidth = 5,
+    nBlocksHeight = 6,
+    fieldWidth = blockWidth * nBlocksWidth,
+    fieldHeight = blockHeight * nBlocksHeight,
+    enemyTracksHeight = [62, 145, 228], //coordinates of enemy tracks
+    playerStartPoint = [200, 405],
```

Comment:
What is 200 and 405?
If I change the playfield parameters how do I calculate those?